### DISCUSSION TITLE
how do I go about opening ranger with a keyboard shortcut

### DISCUSSION THREAD

**Discussion by Merulox** [i3/i3#4863] [Q&A] [answered]
how do I go about opening ranger-- or any TUI with a keyboard shortcut

**Comment by Airblader** [comment, marked as answer]
You create a key binding that launches a terminal and passes the command that terminal should execute. How exactly to do that depends on your terminal, I'm sure it has a man page that will tell you. For example, with alacritty it'd be `alacritty -e command`.
